index.js

```diff
@@ -0,0 +1,75 @@
+const { Assyst } = require('./lib/Assyst.js');
+const config = require('./config.json');
+
+const client = new Assyst(config);
+
+console.log('Starting Assyst');
+
+console.log('Loading commands');
+
+client.loadCommands();
+
+async function checkRepl(msg) {
+    if (!msg.content.startsWith('`') || !msg.content.endsWith('`') ) {
+        return;
+    } if (msg.content.startsWith('```') ) {
+        return;
+    } if (!client.repl.map(i => i.user).includes(msg.author.id) || !client.repl.map(i => i.channel).includes(msg.channel.id) ) {
+        return;
+    }
+    const currentRepl = client.repl.find(j => j.user === msg.author.id && j.channel === msg.channel.id);
+    const contentToSend = msg.content.substr(1, msg.content.length - 2);
+    if (contentToSend === 'exit') {
+        client.repl.splice(client.repl.indexOf(currentRepl, 1) );
+        msg.channel.createMessage(`${client.emotes.success} The REPL session was ended.`);
+        return;
+    }
+    const UtilsFile = require('./src/utils').Utils;
```

Comment:
You will be requiring and instantiating Utils at every usage of this function.
This is also related to my other comment about dealign with Utils instantiation